mobile/Yote/js/resources/user/views/UserLogin.js

```diff
@@ -36,10 +36,8 @@ const UserLogin = () => {
 
   const handleFormSubmit = async (userInfo) => {
     const { payload: result } = await dispatch(sendLogin(userInfo));
-    // adapted from: https://reactrouter.com/web/example/auth-workflow
-    const { from } = location.state || { from: { pathname: "/"} }
     if(result.success) {
```

Comment:
also for future - looks like this one does still use the success: true stuff. thats something that we'll need to update on the server side too though.

Reply:
+1